x/ccv/parent/module.go

```diff
@@ -246,13 +246,16 @@ func (am AppModule) OnChanOpenTry(
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	counterpartyVersion string,
-) (version string, err error) {
-	if err := ValidateParentChannelParams(ctx, am.keeper, order, portID, channelID, ccv.Version); err != nil {
-		return ccv.Version, err
+) (metadata string, err error) {
+
+	if err := ValidateParentChannelParams(
```

Comment:
Update `ValidateParentChannelParams` to not check the version. Currently it checks whether `ccv.Version == ccv.Version`.

Reply:
yup, done